src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -125,7 +125,7 @@ protected function getDumpHeader()
     rxEsc = /([.*+?^${}()|\[\]\/\\])/g,
     idRx = /\bsf-dump-\d+-ref[012]\w+\b/;
 
-doc.documentElement.firstChild.appendChild(refStyle);
+doc.getElementsByTagName('head')[0].appendChild(refStyle);
```

Comment:
This is assuming that the head tag is available to apply the style tag on. Will there be a case that the head tag is missing and it needs the body as fallback?